build/ProjectReunion-BuildFoundation.yml

```diff
@@ -202,6 +223,9 @@ jobs:
 
 - job: BuildMRT
   pool: $(ProjectReunionBuildPool)
+  dependsOn:
+    - VerifyCopyrightHeaders
+  condition: in(dependencies.VerifyCopyrightHeaders.result, 'Succeeded')
```

Comment:
Why are the values in the 'in' conditional different from BuildAnyCPU and Build?

Reply:
I just added the VerifyCopyrightHeaders conditions.  Not sure why BuildMRT already had different conditions.